src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  padding: 20px 0;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    margin-right: 50px;
+  }
+  &__item {
+    list-style: none;
+    margin-left: 20px;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0 22px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $text-color-bold;
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 500;
+    &:hover {
+      color: $hover-color;
+    }
+    &--is-active {
+      color: $hover-color;
+      &:before {
+        content: "";
+        height: 4px;
+        width: 100%;
+        position: absolute;
+        bottom: 1px;
+        left: 0;
+        background-color: $hover-color;
+        border-radius: 8px;
```

Comment:
try to put `padding: 22.5px 0` and in before remove left: 0, and put bottom: 0

Reply:
Thank's, but it didn't help(